app/components/frame-viewer.cjsx

```diff
@@ -144,4 +187,151 @@ module.exports = React.createClass
         width: width ? 0
         height: height ? 0
 
+      viewBoxDimensions:
+        width: width ? 0
+        height: height ? 0
+        x: 0
+        y: 0
+
     @props.onLoad? e, @props.frame
+
+  canZoomOut: ->
+    if @state.frameDimensions.width == @state.viewBoxDimensions.width && @state.frameDimensions.height == @state.viewBoxDimensions.height
```

Comment:
`return @state.frameDimensions.width == @state.viewBoxDimensions.width && @state.frameDimensions.height == @state.viewBoxDimensions.height` should work here instead of testing and returning true if true.